app/js/directives/snVelocity.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
```

Comment:
I don't think it's necessary to use namespace for filenames you can just call the files `velocity.js` and `velocity-group.js`

http://stackoverflow.com/questions/7273316/what-is-the-javascript-filename-naming-convention